helm/hpcc/values.yaml

```diff
@@ -188,6 +188,10 @@ secrets:
     ## For example, to set the secret associated with the azure storage account "mystorageaccount" use
     ##azure-mystorageaccount: storage-myazuresecret
 
+  authn: {}
+    ## Category for LDAP Admin username/password
+    MyLDAPServerKey: MyLDAPServerValue
```

Comment:
This example should be removed or commented out.  If someone uses this and the secret doesn't exist their pods won't run.. so this should only be added when the user really wants to use ldap.